gobdd_go1_16.go

```diff
@@ -0,0 +1,63 @@
+//go:build go1.16
+// +build go1.16
+
+package gobdd
+
+import (
+	"fmt"
+	"io"
+	"io/fs"
+)
+
+// WithFeaturesFS configures a filesystem and a pattern (regexp) where features can be found.
+// An empty path defaults to "*.feature"
+func WithFeaturesFS(fs fs.FS, pattern string) func(*SuiteOptions) {
```

Comment:
Should we add a separate function that doesn't require a pattern? Might be slightly better than having to pass an empty string as a second parameter.